src/main/java/com/twilio/type/OutboundPrefixPriceWithOrigin.java

```diff
@@ -0,0 +1,104 @@
+package com.twilio.type;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Outbound prices for prefixes with origin.
+ *
+ * <p>
+ *     For more information see:
+ *     <a href=https://www.twilio.com/docs/api/pricing/voice#outbound-prefix-price>Pricing docs</a>
+ *     TODO TODO TODO
```

Comment:
?

Reply:
Yeah, is that online documentation generated automatically? That link points to version 1 of this endpoint. Not sure what to put there...

Reply:
Yeah the docs are autogenerated based on the docstrings -- they'll get published when we do another build of the library (on Friday). So I think it's fine to just have the link broken for now -- you could double-check that it's correct after we publish the library on Friday or Monday.